classes/log/store.php

```diff
@@ -68,6 +78,17 @@ public function __construct(log_manager $manager) {
      *
      */
     protected function is_event_ignored(event_base $event) {
+
+        if ((!CLI_SCRIPT or PHPUNIT_TEST) and !$this->logguests) {
```

Comment:
Yeah please continue with the change, our style is likely to be different to the Moodle core.